staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -27,10 +27,12 @@ const (
 
 	// JobTrackingFinalizer is a finalizer for Job's pods. It prevents them from
 	// being deleted before being accounted in the Job status.
-	// The apiserver and job controller use this string as a Job annotation, to
-	// mark Jobs that are being tracked using pod finalizers. Two releases after
-	// the JobTrackingWithFinalizers graduates to GA, JobTrackingFinalizer will
-	// no longer be used as a Job annotation.
```

Comment:
if JobTrackingWithFinalizers is graduating to GA in 1.26, can we make this specific now? (e.g. "In 1.28+, two releases after JobTrackingWithFinalizers graduates to GA, this will no longer be used as a Job annotation.")